pylua/test/websocket_test1.lua

```diff
@@ -0,0 +1,17 @@
+require('pylua/network')
+
+function lua_test1()
+	client = WebSocketClient.WebSocketClient()
+	client.open("ws://localhost:8087/")
+	client.write("lua_test1")
+	s = client.read()
+
+	print('Message from server:')
+	print('=====================')
+	print(s)
+	print('=====================')
+
+	client.close()
+end
+
+lua_test1()
```

Comment:
@OHerasym, try imagine more test cases. One looks not so good. I can prose the following to add: create two clients sending different data (check that they will receive their own data), connect\send\disconnect\verify in a loop (2 - 3 iterations). may be you can add more.